tests/page-objects/reports/reports.po.js

```diff
@@ -82,30 +91,70 @@ module.exports = {
     expect(element.all(by.css('#reports-list li[data-record-id="' + savedUuids[1] + '"]')).count()).toBe(1);
   },
 
+  deleteSelectedReportsNative: async (savedUuids) => {
+    await element(by.css('.action-container .detail-actions .delete-all')).click();
+    const confirmButton = element(by.css('.btn.submit.btn-danger'));
+    await helper.waitElementToBeClickable(confirmButton, 5000);
+    await confirmButton.click();
+    await helper.waitElementToBeClickable(confirmButton, 5000);
+    await confirmButton.click();
+    await helper.waitElementToPresent(element(by.css('#reports-list li')), 30000);
+    // make sure the reports are deleted
+    expect(await reportsByUUID(savedUuids[1]).count()).toBe(1);
+  },
+
+
   deselectAll: () => {
+    utils.deprecated('deselectAll','deselectAllNative');
     element(by.css('.action-container .deselect-all')).click();
     expect(element(by.css('#reports-content .selection-count > span:first-child')).isDisplayed()).toBeFalsy();
     expect(element.all(by.css(reportBody)).count()).toBe(0);
   },
 
+  deselectAllNative: async () => {
+    await element(by.css('.action-container .deselect-all')).click();
+    expect(await element(by.css('#reports-content .selection-count > span')).isPresent()).toBeFalsy();
+    expect(await element.all(by.css(reportBody)).count()).toBe(0);
```

Comment:
These `expect` checks are effectively "test logic" and seeing them here in a Page Object file is unexpected (no pun intended). Right now the focus is on getting things migrated, so it's ok to keep it as-is, but over time let's work to keep all the `expect` calls in the tests themselves and keep Page Objects as helpers for just doing the basic reading of things on the page.